bukuserver/templates/bukuserver/home.html

```diff
@@ -9,48 +9,47 @@
 
 {% block menu_links %}
 {{ super() }}
-<form class="navbar-form navbar-right" action="{{ url_for('bookmark.index_view') }}" method="GET">
-  <div class="form-group">
+<form class="form-inline navbar-right" style="gap:.5ex" action="{{ url_for('bookmark.index_view') }}" method="GET">
+  <div class="d-inline-block align-middle">
```

Comment:
`navbar-form` was removed in Bootstrap4; and `form-group` doesn't have the desired effect here.